sphinxcontrib/images.py

```diff
@@ -265,13 +265,15 @@ def download_images(app, env):
                       .format(src, dst))
 
 
-def configure_backend(app):
+def configure_backend(app, main_config):
     global DEFAULT_CONFIG
 
     config = copy.deepcopy(DEFAULT_CONFIG)
     config.update(app.config.images_config)
     app.config.images_config = config
```

Comment:
I would probably use the `main_config` directly since it's provided, but I'm not sure if there is a good reason why the hook has both app and the config though.


```suggestion
def check_config(app, config):                                                 
    '''Ensure all config values are defined'''                                 
                                                                               
    merged = copy.deepcopy(DEFAULT_CONFIG)                                     
    merged.update(config.images_config)                                        
    config.images_config = merged 
```